scripts/processToFile.js

```diff
@@ -22,5 +24,29 @@ if(matches) {
 }
 
 var newFileName = mainProcessFile.replace('.template', '.js');
+
+// Look for <<param>>, see if there is a index attached
+// and replace that string with the corresponding argument.
+//   <<param>> will be replaced w/ process.argv[PARAM_START_INDEX]
+//   <<param2>> will be replaced w/ process.argv[PARAM_START_INDEX+2]
+regex = new RegExp('<<(param.*)>>', 'g');
+matches = mainProcess.match(regex);
+if(matches) {
+    var param_max = process.argv.length - PARAM_START_INDEX;
```

Comment:
what about this, it is easier:

```
for(var i = 0; i < param_max; i++) {
  mainProcess = mainProcess.replace('<<param' + i + '>>', process.argv[PARAM_START_INDEX + i]);
}
```

it is not necessary to find the matchs of 'param.*'. The replace function doesn't do anything if it doesn't find the token.